src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.search-form__lable {
+  display: block;
+  position: relative;
+}
+
+.search-form__lable::before {
+  content: "";
+  position: absolute;
+
+  background-image: url("./images/Search.svg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.search-form__field {
+  display: block;
+  font-family: inherit;
+  width: 100%;
+  outline: none;
+  border-radius: 4px;
+  color: inherit;
+  font-size: inherit;
+  font-weight: inherit;
```

Comment:
Do we really need to manually specify styles with the `inherit` value here? CSS styles are automatically followed.

Reply:
if i remove these rules the styles aren't followed except font-weight so i removed it